ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -189,7 +196,7 @@ def _map_sample(self, sample, exclude_fields):
 class DatasetToolArgs:
     # HF source dataset parameters
     dataset_name: str = simple_parsing.field(alias="-d")
-    dataset_subset: Optional[str] = simple_parsing.field(default=None, alias="-S")
+    dataset_subset: Optional[str] = simple_parsing.field(default="default", alias="-S")
```

Comment:
What happens if the dataset only has one subset/config whose name is not `default`, when calling `datasets.load_dataset` without specifying the subset/config?